src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -58,4 +68,86 @@ static ProcessMetrics()
     public ProcessMetrics(ProcessInstrumentationOptions options)
     {
     }
+
+    private class InstrumentsValues
+    {
+        private const int TicksPerSecond = 10 * 7;
+        private readonly Diagnostics.Process currentProcess = Diagnostics.Process.GetCurrentProcess();
+
+        private double? memoryUsage;
+        private double? virtualMemoryUsage;
+        private double? cpuTime;
+        private double? cpuUtlization;
+
+        private DateTime lastCollectionTimestamp = DateTime.Now;
```

Comment:
Use `DateTime.UtcNow` instead of `DateTime.Now`. It is much faster.

Reply:
In addition, these are not monotonic, I think for duration we should use something that measures elapsed time rather than wall clock.